consensus/core/src/universal_committer.rs

```diff
@@ -43,11 +43,26 @@ impl UniversalCommitter {
 
         // Try to decide as many leaders as possible, starting with the highest round.
         let mut leaders = VecDeque::new();
+
+        let last_round = if self
+            .context
+            .protocol_config
+            .mysticeti_disable_multi_leader_per_round_in_committer()
```

Comment:
Can we combine this feature flag with the `NUM_LEADERS_PER_ROUND` constant?

Reply:
Good idea!!